src/nickel_error.rs

```diff
@@ -27,10 +28,10 @@ impl NickelError {
     }
 }
 
-#[deriving(Show)]
+#[derive(Show)]
 pub enum NickelErrorKind {
     // FIXME: Should probably re-export http::status::Status
     ErrorWithStatusCode(Status),
-    UserDefinedError(int, String),
+    UserDefinedError(usize, String),
```

Comment:
This is a semantic change, but I'm mostly alright with it. The error system still needs a whole revamp to match modern conventions.